src/fastalloc/tests.rs

```diff
@@ -140,18 +140,20 @@ impl RealFunction {
     }
 }
 
-fn mach_env(no_of_regs: usize) -> MachineEnv {
+fn mach_env(no_of_regs: usize) -> MachineEnv<'static> {
     MachineEnv {
         preferred_regs_by_class: [
-            (0..no_of_regs)
-                .map(|no| PReg::new(no, RegClass::Int))
-                .collect(),
-            vec![],
-            vec![],
+            Vec::leak(
```

Comment:
We definitely can't leak while fuzzing -- can we instead have a few static `MachineEnv`s in scope, and pick one of them? We don't necessarily need to fuzz with arbitrary `no_of_regs`, just with a few values that are interesting -- the two extremes of its range and a few in the middle, for example.